contracts/collections/sg721-metadata-onchain/Cargo.toml

```diff
@@ -38,7 +38,10 @@ sg721           = { workspace = true }
 sg721-base      = { workspace = true, features = ["library"] }
 sg-metadata     = { workspace = true }
 sg-std          = { workspace = true }
+cw721-base-016  = { workspace = true }
```

Comment:
is this necessary? can you just add this to tests?

Reply:
Seems like this is a redundant package now.